### PR TITLE
Refactor/rename merge parent into current

### PR DESCRIPTION
**Pull request by azdrojowa123**

### CODE REVIEW

File: frontend/base/src/main/resources/GitMacheteBundle.properties [hunk 1]
```diff
@@ -318,8 +318,8 @@ action.GitMachete.FastForwardMergeCurrentBranchToParentAction.GitMacheteToolbar.
 action.GitMachete.FastForwardMergeCurrentBranchToParentAction.description=Fast-forward merge current branch to parent
 
 
-action.GitMachete.MergeParentIntoCurrentBranchAction.text=Git Machete: Merge Parent into Current Branch
-action.GitMachete.MergeParentIntoCurrentBranchAction.GitMacheteToolbar.text=Merge Parent into Current Branch
+action.GitMachete.MergeParentIntoCurrentBranchAction.text=Git Machete: Sync to Parent by Merge
+action.GitMachete.MergeParentIntoCurrentBranchAction.GitMacheteToolbar.text=Sync to Parent by Merge
```
Comment (PawelLipski) [inline frontend/base/src/main/resources/GitMacheteBundle.properties:322]: While we're here... are the descriptions generally consistent for other actions, esp. Rebase? They don't need to be identical, just it should be clear that the action in toolbar and in context menu correspond to the same thing!

**Review by mkondratek — APPROVED** [review]

File: frontend/base/src/main/resources/GitMacheteBundle.properties (hunk 1 above)
Comment (PawelLipski) [reply frontend/base/src/main/resources/GitMacheteBundle.properties:322]: Plus as discussed, pls document:
* `...` usage
* context menu/toolbar action name consistency
in contributing guidelines

Comment (mkondratek) [reply frontend/base/src/main/resources/GitMacheteBundle.properties:322]: good point. I see 
`action.GitMachete.RebaseCurrentBranchOntoParentAction.description=Rebase current branch onto parent`
in particular. should be updated too

**Review by mkondratek — CHANGES_REQUESTED** [review]

File: frontend/base/src/main/resources/GitMacheteBundle.properties (hunk 1 above)
Comment (azdrojowa123) [reply frontend/base/src/main/resources/GitMacheteBundle.properties:322]: On this icon bar sometimes it is mention that `do sth ... on current branch` and sometimes not e.q. `Fast Forward merge Current branch into parent` and `Sync to Parent by rebase` (if we want to stick to the convention, it should probably also be information that current branch ... or delete this information from `Fast forward merge ...`)

Comment (azdrojowa123) [reply frontend/base/src/main/resources/GitMacheteBundle.properties:322]: This situation is only with `Sync to Parent by rebase` and` sync to parent by merge` (no current branch information )

**Review by PawelLipski — APPROVED** [review]

**Pull request closed without merging**